### DISCUSSION TITLE
Radio

### DISCUSSION THREAD

**Discussion by mrvietcong12345** [eritislami/evobot#1072] [Q&A]
I try to make a command "rock" that will start playing a rock playlist or my custom playlist. But I got no idea how to make it.

**Comment by Velosofy** [comment]
I guess you can do it in a janky way by:

Let's say I have a config.js var of "rock" with a youtube playlist as a string
So I would do:
- Modify the `play.js` to respond if the user type `play rock`, it'll play your predefined playlist

Something like this as an example,

play.js
```js
const rock = "a youtube playlist url"

    if (args[0] === "rock")
      args[0] = rock;
```
